contracts/src/v0.8/dev/interfaces/OCR2DROracleInterface.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+/**
+ * @title OCR2DR oracle interface.
+ */
+interface OCR2DROracleInterface {
+  function sendRequest(
+    address sender,
+    uint256 nonce,
+    uint256 subscriptionId,
+    bytes calldata data
+  ) external;
+
+  function cancelRequest(bytes32 requestId) external;
+
+  function fulfillRequest(
+    bytes32 requestId,
+    address callbackAddress,
+    bytes calldata response,
+    bytes calldata err
+  ) external;
+}
```

Comment:
Is there only 1 instance of this oracle contract that is shared amongst all DON nodes, or does each node deploy an instance of this contract?

Reply:
One per DON, I hope...?

Reply:
Let's try discussing this today internally, looks like this is unclear for many people.